liquibase-core/src/test/java/liquibase/statement/InsertExecutablePreparedStatementTest.java

```diff
@@ -0,0 +1,145 @@
+package liquibase.statement;
+
+import liquibase.change.ColumnConfig;
+import liquibase.changelog.ChangeSet;
+import liquibase.database.Database;
+import liquibase.database.PreparedStatementFactory;
+import liquibase.database.core.MSSQLDatabase;
+import liquibase.database.jvm.JdbcConnection;
+import liquibase.resource.ResourceAccessor;
+import liquibase.structure.core.Column;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.*;
+
+import java.sql.PreparedStatement;
+import java.util.ArrayList;
+
+import static java.util.Arrays.asList;
+import static org.mockito.MockitoAnnotations.initMocks;
+import static org.mockito.Mockito.*;
+
+public class InsertExecutablePreparedStatementTest {
+    @Mock
+    private ChangeSet changeSet;
+
+    @Mock
+    private ResourceAccessor resourceAccessor;
+
+    @Mock
+    private JdbcConnection connection;
+
+    @Mock
+    private PreparedStatement ps;
+
+    private Database database;
+
+    private PreparedStatementFactory preparedStatementFactory;	
+
+    @Before
+    public void setUp() throws Exception {
+        initMocks(this);
+        when(connection.prepareStatement(any(String.class))).thenReturn(ps);
+        database = new MSSQLDatabase();
+        preparedStatementFactory = new PreparedStatementFactory(connection);
+    }
+	
+	@Test
+	public void testExecuteWithClobAndComputedValue() throws Exception {
+        // given
+        InsertExecutablePreparedStatement statement = new InsertExecutablePreparedStatement(
+                        database,
+                        null,
+                        null,
+                        "DATABASECHANGELOG",
+                        new ArrayList<ColumnConfig>(asList(
+                                new ColumnConfig()
+                                        .setName("MD5SUM")
+                                        .setValue("7:e27bf9c0c2313160ef960a15d44ced47"),
+                                new ColumnConfig()
+                                        .setName("DATEEXECUTED")
+                                        .setValueDate("GETDATE()"))),
+                        changeSet,
+                        resourceAccessor);
+
+        // when
+        statement.execute(preparedStatementFactory);
+
+        // then
+        verify(connection).prepareStatement("INSERT INTO DATABASECHANGELOG(MD5SUM, DATEEXECUTED) VALUES(?, GETDATE())");
+		verify(ps).setString(1, "7:e27bf9c0c2313160ef960a15d44ced47");
+		verify(ps, never()).setNull(eq(2), anyInt());
+	}
+
+//    @Test
```

Comment:
Why did you end up commenting out these new tests?

Reply:
I just copied UpdateExecutablePreparedStatementTest and changed the first test to demonstrate the bug I wanted to fix. The second test is also commented there, I didn't even think about that, sorry.